components/Form.js

```diff
@@ -9,10 +9,15 @@ export default function Form({ currentTrips, setCurrentTrips }) {
     const newTripObject = {
       tripName: data.name,
```

Comment:
Maybe you can give your `newTripObject` an id key to later use it as a valid unique key. Can it happen, that some trips are called the same? But are just in another time span with another tripUser?